bot/code_review_bot/tasks/upload.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
```

Comment:
The name of the file, `upload.py`, is not great. I can't think of a super nice name, but `docs.py` or `docupload.py` would definitely be better.

@sylvestre any idea?